jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -333,11 +333,20 @@ MlirToJlmConverter::ConvertBitBinaryNode(
 
   auto type = mlirOperation.getResult(0).getType();
 
-  if (!type.isa<::mlir::IntegerType>())
+  size_t width = 0;
+  if (type.isa<::mlir::IntegerType>())
+  {
+    auto integerType = type.cast<::mlir::IntegerType>();
+    width = integerType.getWidth();
+  }
+  else if (type.isIndex())
+  {
+    width = 32;
```

Comment:
Why 32? Ideally we would get the size of `uintptr` from somewhere central, but I would at least expect it to be 64, if we are hard coding it for now

Reply:
The reason is that the constant index for the GepOp seems to be hard coded to 32, so I stuck with the same size for these.